lib/pbench/test/unit/server/test_user_create.py

```diff
@@ -1,7 +1,7 @@
 from click.testing import CliRunner
 import responses
 
-from pbench.cli.server import user_create
+from pbench.cli.server.user_management import user_create
```

Comment:
Unit tests for the other `user` functions?  (Should you rename this module to something like `test_user_management` and add them here?)

Reply:
Yes I am updating the unit test file to test all the commands